### DISCUSSION TITLE
Adding @index and query param

### DISCUSSION THREAD

**Discussion by jhavelik** [mockoon/mockoon#620] [Help] [answered]
Can I somehow use [add feature](https://mockoon.com/docs/latest/templating/mockoon-helpers/#add) where I'm adding `@index` (from [repeat](https://mockoon.com/docs/latest/templating/mockoon-helpers/#repeat)) and `queryParam`, I have tried following options and it didn't worked:
`{{add @index queryParam 'offset' '0'}}`
`{{add @index {{queryParam 'offset' '0'}}}}`

But this seems to work:
`{{add @index 10}}`

I'm open to suggestions (even workarounds), This would help me with mocking to be very close to real life results.

**Comment by 255kb** [comment, marked as answer]
You must use parenthesis to prioritize, otherwise Handlebars won't be able to use your params in the correct order: 
`{{add @index (queryParam 'offset' '0')}}`
Also, you can never use `{{ }}` inside `{{ }}`.
